rov/movement/thruster/Force_Limiter.py

```diff
@@ -0,0 +1,105 @@
+from math import floor
```

Comment:
If this class is not used anymore, delete it. We can always get it again through git anyway.